source/simulator/assembly.cc

```diff
@@ -280,22 +282,23 @@ namespace aspect
 
           local_dof_indices (finite_element.dofs_per_cell),
 
-          phi_field (advection_element.dofs_per_cell, Utilities::signaling_nan<double>()),
-          grad_phi_field (advection_element.dofs_per_cell, Utilities::signaling_nan<Tensor<1,dim> >()),
-          old_velocity_values (quadrature.size(), Utilities::signaling_nan<Tensor<1,dim> >()),
-          old_old_velocity_values (quadrature.size(), Utilities::signaling_nan<Tensor<1,dim> >()),
-          old_pressure (quadrature.size(), Utilities::signaling_nan<double>()),
-          old_old_pressure (quadrature.size(), Utilities::signaling_nan<double>()),
-          old_pressure_gradients (quadrature.size(), Utilities::signaling_nan<Tensor<1,dim> >()),
-          old_old_pressure_gradients (quadrature.size(), Utilities::signaling_nan<Tensor<1,dim> >()),
-          old_strain_rates (quadrature.size(), Utilities::signaling_nan<SymmetricTensor<2,dim> >()),
-          old_old_strain_rates (quadrature.size(), Utilities::signaling_nan<SymmetricTensor<2,dim> >()),
-          old_temperature_values (quadrature.size(), Utilities::signaling_nan<double>()),
-          old_old_temperature_values(quadrature.size(), Utilities::signaling_nan<double>()),
-          old_field_grads(quadrature.size(), Utilities::signaling_nan<Tensor<1,dim> >()),
-          old_old_field_grads(quadrature.size(), Utilities::signaling_nan<Tensor<1,dim> >()),
-          old_field_laplacians(quadrature.size(), Utilities::signaling_nan<double>()),
-          old_old_field_laplacians(quadrature.size(), Utilities::signaling_nan<double>()),
+          phi_field (advection_element.dofs_per_cell),
```

Comment:
Definitely. Of course, this suggests the question whether there are other parts of the patch that also overwrite recently introduced functionality? Did you read through your patch again?